tests/testScripts/test_landing.py

```diff
@@ -2,29 +2,96 @@
 
 import config
 import pytest
+import time
 
 """
     Testing the 'Landing' page
 """
 
 
-def test_landing_login(admin_setup):
+def test_landing_login_with_correct_credentials(admin_setup):
     """
          Verify that user has the ability to login in as an Admin.
     """
-    expected_result = "AlexTestEngineer"
+    expected_result = "InnaName"
     admin_setup.landing.find_event_btn.click_btn_by_css()
     assert expected_result == admin_setup.landing_authorized_user.get_user_name(), \
         "username results are not the same as expected"
+    time.sleep(10)
 
 
-def test_landing_registration(app):
+def test_landing_registration_with_correct_credentials(app):
     """
         Verify that the user has the ability to register a new account.
     """
-    expected_result = "Your registration was successfully. Please confirm your email."
+    expected_result = "Your register was successfull. Please confirm your email."
     app.landing.go_to_site()
-    app.landing.sign_up_btn.click_btn_by_css()
+    app.landing.sign_in_up_btn.click_btn_by_css()
     app.modal.registration(config.ADMIN_EMAIL, config.ADMIN_PASS)
     assert expected_result == app.modal.get_success_register_text(), \
         "alert message is not the same as expected"
+
```

Comment:
don't change the functionality and purpose of this method. Create one more instead